iotedgedev/module.py

```diff
@@ -0,0 +1,49 @@
+
+import os
+import json
+
+
+class Module(object):
+    def __init__(self, output, utility, module_json_file):
+        self.output = output
+        self.utility = utility
+        self.module_json_file = module_json_file
+
+        self.module_language = "csharp"
+        self.file_json_content = None
+        self.load_module_json()
+
+    def load_module_json(self):
+        if os.path.exists(self.module_json_file):
+            try:

```

Comment:
We should do some more schema validation here, I'll add a task for a future update.